### PR TITLE
Core validation error enum updates

### PR DESCRIPTION
**Pull request by mikes-lunarg**

### CODE REVIEW

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -7958,7 +7981,7 @@ TEST_F(VkLayerTest, PSOLineWidthInvalid) {
 TEST_F(VkLayerTest, NullRenderPass) {
     // Bind a NULL RenderPass
     m_errorMonitor->SetDesiredFailureMsg(VK_DEBUG_REPORT_ERROR_BIT_EXT,
-                                         "You cannot use a NULL RenderPass object in vkCmdBeginRenderPass()");
+                                         "vkCmdBeginRenderPass: required parameter pRenderPassBegin specified as NULL");
```
Comment (mark-lunarg) [inline tests/layer_validation_tests.cpp]: Probably doesn't matter this time but this change is unrelated to the others and would be more clear by being in a separate commit.

**Comment by mikes-lunarg** [comment]
Pushed to master
